.github/workflows/covector-version-or-publish.yml

```diff
@@ -0,0 +1,47 @@
+name: version or publish
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  version-or-publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 65
+    outputs:
+      change: ${{ steps.covector.outputs.change }}
+      commandRan: ${{ steps.covector.outputs.commandRan }}
+      successfulPublish: ${{ steps.covector.outputs.successfulPublish }}
```

Comment:
we don't actually care about these right?
```suggestion
```

Reply:
I don't think so. Copy pasted it without thinking about it :P